app/src/main/java/org/sopt/and/presentation/authentication/SignInScreen.kt

```diff
@@ -55,41 +128,44 @@ fun SignInScreen(
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
         SignInTopBar()
-
-        CustomTextField(
+        AuthTextField(
             modifier = Modifier
                 .padding(top = 48.dp)
                 .height(56.dp),
             value = emailInput,
             hint = stringResource(R.string.sign_in_email_hint),
-            onValueChange = {
-                if (it.length <= Constants.MAX_EMAIL) emailInput = it
-            },
+            onValueChange = { onEmailChange(it) },
```

Comment:
이 경우에도 앞선 리뷰와 똑같은 경우라면 함수를 그냥 전달하면 됩니다! 
```suggestion
            onValueChange = onEmailChange,
```